package.json

```diff
@@ -13,7 +13,7 @@
 		"url": "https://github.com/dojo/shim.git"
 	},
 	"scripts": {
-		"prepublish": "grunt dist",
+		"prepublish": "grunt peerDepInstall dist",
```

Comment:
> but I remember that there was some push back

Hopefully not from me and now I am changing my mind. 😕 I guess there is no harm in this way, because the biggest guards is making sure the package as a whole is up-to-date and compiles successfully before being published.